integration/flink/flink117/build.gradle

```diff
@@ -0,0 +1,139 @@
+/*
+/* Copyright 2018-2023 contributors to the OpenLineage project
+/* SPDX-License-Identifier: Apache-2.0
+*/
+
+plugins {
+    id 'java'
+    id 'java-library'
+    id 'maven-publish'
+    id 'signing'
+    id 'jacoco'
+    id "com.adarshr.test-logger" version "3.2.0"
+    id 'com.diffplug.spotless' version '6.25.0'
+    id "com.github.johnrengelman.shadow" version "8.1.1"
+    id "pmd"
+    id 'io.franzbecker.gradle-lombok' version '5.0.0'
+}
+
+repositories {
+    mavenLocal()
+    mavenCentral()
+    maven {
+        url = "https://packages.confluent.io/maven/"
+    }
+}
+
+pmd {
+    consoleOutput = true
+    toolVersion = "6.46.0"
+    rulesMinimumPriority = 5
+    ruleSetFiles = rootProject.files("pmd-openlineage.xml")
+    ruleSets = []
+    ignoreFailures = true
+}
+
+pmdMain {
+    dependsOn shadowJar
+    reports {
+        html.required = true
+    }
+}
+
+configurations {
+    lombok
+}
+
+java {
+    sourceCompatibility = JavaVersion.VERSION_11
+    targetCompatibility = JavaVersion.VERSION_11
+}
+
+group 'io.openlineage.flink'
+
+sourceSets {
+    main.java.srcDirs = ["src/main/java"]
+    test.java.srcDirs = ["src/test/java"]
+}
+
+task sourceJar(type: Jar) {
+    archiveClassifier = 'sources'
+    from sourceSets.main.allJava
+}
+
+task javadocJar(type: Jar, dependsOn: javadoc) {
+    archiveClassifier = 'javadoc'
+    from javadoc.destinationDir
+}
+
+def commonTestConfiguration = {
+    forkEvery 1
+    maxParallelForks 3
+    testLogging {
+        events "passed", "skipped", "failed"
+        showStandardStreams = true
+    }
+    systemProperties = [
+            'junit.platform.output.capture.stdout': 'true',
+            'junit.platform.output.capture.stderr': 'true',
+            '.flink.jar': "${archivesBaseName}-${project.version}.jar",
+    ]
+    classpath =  project.sourceSets.test.runtimeClasspath
+}
+
+test {
+    configure commonTestConfiguration
+    useJUnitPlatform {
+        excludeTags 'integration-test'
+    }
+}
+
+ext {
+    flinkVersion = project.getProperty('flink.flink117.version')
+    assertjVersion = '3.25.3'
+    junit5Version = '5.10.2'
+    mockitoVersion = '5.2.0'
+    jdbcVersion = '3.1.1-1.17'
+}
+
+dependencies {
+    implementation(project(path: ":shared"))
+    api "io.openlineage:openlineage-java:${project.version}"
+    api "io.openlineage:openlineage-sql-java:1.8.0"
```

Comment:
If I change to ${project.version}, integration test will fail due to fail to load the native libs.

Reply:
This is working when running on CI, please move it back to `${project.version}` once it passes locally.